tests/user.test.js

```diff
@@ -0,0 +1,61 @@
+const request = require("supertest");
+const axios = require('axios');
+const { createApp } = require("../app");
+const AppDataSource = require("../src/models/data-source");
```

Comment:
```suggestion
const appDataSource = require("../src/models/data-source");
```
appDataSource는 new를 통해 만들어진 instance이고 class 자체는 아니기에 범용적인 convention에 따라 수정해주세요~~